src/services/bundle_repository.js

```diff
@@ -111,7 +132,7 @@ export default class BundleRepository {
 
   async isBundleSheltered(bundleId) {
     const repository = await this.getBundleRepository(bundleId);
-    return repository ? repository.status === BundleStatusStates.sheltered : false;
+    return repository ? repository.status === BundleStatuses.sheltered : false;
```

Comment:
Why `false` if there is no `repository` field?

Reply:
We don't know if it's sheltered looking only on status, so yes